src/chrome-helper.ts

```diff
@@ -482,6 +483,14 @@ export const convertUrlParamsToLaunchOpts = (
   const keepalive = _.isNaN(parsedKeepalive) ? undefined : parsedKeepalive;
   const parsedIgnoreDefaultArgs = parseIgnoreDefaultArgs(urlParts.query);
 
+  const playwrightVersion = (() => {
+    const uAgent = req.headers['user-agent'];
+    if (!uAgent || !uAgent.startsWith('Playwright/')) return undefined;
+
+    const split = uAgent.split('Playwright/')[1].split('.');
```

Comment:
We might be able to regex the version out a bit better than doing `.split` calls. Can you remind me an example of what the user-agent string is?

Reply:
Ah, I totally forgot about RegEx existence. This is the string:
`Playwright/1.25.1 (x64; windows 10.0) node/16.14`

Reply:
Used RegEx `(?<=Playwright\/)(\d+(\.\d+))`
https://regex101.com/r/y1LYDA/1